### PR TITLE
Add Simple

### PR DESCRIPTION
**Pull request by KevinBongart**
Adding [Simple](https://www.simple.com/) to the finance section. Great bank, but they need to provide better security to critical data!

### CODE REVIEW

**Comment by computmaxer** [comment]
@KevinBongart This PR has merge conflicts, please pull in the latest master to resolve.

**Comment by KevinBongart** [comment]
@computmaxer Done!

File: _data/main.yml [hunk 1]
```diff
@@ -138,6 +138,12 @@ sections:
           img: capitalone360.png
           tfa: No
 
+        - name: Simple
+          url: https://www.simple.com/
+          twitter: simple
+          img: mint.png
```
Comment (ilyakatz) [inline _data/main.yml]: simple.png perhaps? I did the same thing :)

Comment (KevinBongart) [inline _data/main.yml]: Thanks! Fixed in 9911fbd

**Comment by computmaxer** [comment]
:+1:

**Pull request merged**